config/local/system.theme.yml

```diff
@@ -0,0 +1,4 @@
+admin: seven
```

Comment:
I'm thinking this will change to be adminimal_theme in due course, but fine with me to leave it at seven for now